Orange/widgets/model/tests/test_owgradientboosting.py

```diff
@@ -220,37 +223,39 @@ def test_default_parameters_cls(self):
         booster = XGBRFClassifier()
         model = booster(data)
         params = model.skl_model.get_params()
+        booster_params = json.loads(model.skl_model.get_booster().save_config())
```

Comment:
Hahaha, this is a interesting way to get the parameters. I guess this is the only way now. I think we should make sure this also works with the oldest supported xgboost. I see that the "oldest" tests do not test that. Could you also ensure that that gets tested?

Reply:
Thanks!